scripts-components/shapes/package.json

```diff
@@ -0,0 +1,43 @@
+{
+	"name": "@mangoweb/shapes",
+	"version": "0.0.3",
+	"description": "Injects svg with symbols.",
+	"main": "lib/index.js",
+	"types": "lib/index.d.ts",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"build": "tsc",
+		"eslint:lint": "eslint \"src/**/*.{ts,tsx}\" ",
+		"eslint:fix": "eslint --fix \"src/**/*.{ts,tsx}\" ",
+		"prepublish": "npm run build"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/manGoweb/npm/tree/master/scripts-base"
+	},
+	"author": "manGoweb <info@mangoweb.cz> (https://mangoweb.cz)",
+	"license": "MIT",
+	"devDependencies": {
+		"@mangoweb/eslint-config-ts": "^0.0.0",
+		"@mangoweb/prettier-config": "^0.0.0",
+		"@typescript-eslint/eslint-plugin": "^1.9.0",
+		"@typescript-eslint/parser": "^1.9.0",
+		"eslint": "^5.16.0",
+		"eslint-config-prettier": "^5.0.0",
+		"eslint-plugin-prettier": "^3.1.0",
+		"eslint-plugin-react": "^7.13.0",
+		"eslint-plugin-react-hooks": "^1.6.0",
+		"prettier": "^1.17.0",
+		"typescript": "^3.5.1"
+	},
+	"prepare": "npm run build",
+	"publishConfig": {
+		"access": "public"
+	},
+	"dependencies": {
+		"isinview": "^2.1.0"
```

Comment:
Proč to má `isinview` dependency?

Reply:
Dobrý postřeh. Opraveno. Díky.

(Jak je to ve stejném repu s `InView` komponentou a sdílí stejnou strukturu, tak jsem to přehlédl.)